lib/common.sh

```diff
@@ -2,33 +2,40 @@
 if [ -z "${COMMON_INCLUDED:-}" ]; then
     COMMON_INCLUDED=YES
 
-    # Colorful output.
+    # Console print with dashes as separator
+    function print_separator {
+      text="$1"
+
+      echo -e "--------------------\n${text}\n--------------------"
+    }
+
+    # Colorful output
     function greenprint {
         echo -e "\033[1;32m[$(date -Isecond)] ${1}\033[0m"
     }
 
-    # filter out resources older than X hours
+    # Filter out resources older than X hours
     HOURS_BACK="${HOURS_BACK:-6}"
     DELETE_TIME=$(date -d "- $HOURS_BACK hours" +%s)
 
-    export DRY_RUN="false"
+    export DRY_RUN='false'
```

Comment:
IIRC in bash we'd prefer double quotes. Is there any particular reason to change them into single quotes?

Reply:
There's no particular reason, just a preference. I usually use double quotes only if needed, but if we agree on using double quotes everywhere that's totally fine for me.

Reply:
OK got it. 

I think ShellCheck will eliminate the question around preferences. Whatever it says is good that's what we use.

Reply:
I think ShellCheck doesn't complain about single vs. double quotes. I ran it locally and it didn't complain at all. So I think it's a matter of defining it as code style convention for this project (we can add it to README.md).

I just found this page that summarizes the differences quite well so that we can better decide:
* https://phoenixnap.com/kb/bash-single-vs-double-quotes

So, sometimes either one or the other is strictly needed, depending on what you want to do.

I think we can either agree on a sort of code style convention, or just allow both of them, based on developer's preference. (I prefer the first option but I understand it can be annoying sometimes).

Reply:
We can add `shfmt` in a separate PR to end all style discussions once and forever. :D

https://github.com/mvdan/sh

Reply:
Oh yes, this is a great idea!